### PR TITLE
Fix typos: creats -> creates

### PR DESCRIPTION
**Pull request by mooncak**
Signed-off-by: mooncake <xcoder@tenxcloud.com>

Fix typos: creats -> creates

### CODE REVIEW

**Comment by idealhack** [comment]
Hi, thanks for your contribution. Since you have opened more than one PRs like this, please considering combine multiple trivial fixes in one PR to save reviewers time. Read more [here](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#10-trivial-edits).

**Comment by mooncak** [comment]
> Hi, thanks for your contribution. Since you have opened more than one PRs like this, please considering combine multiple trivial fixes in one PR to save reviewers time. Read more [here](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#10-trivial-edits).

ok

**Comment by mooncak** [comment]
@idealhack @nikhita Hi, already merged into this PR.

File: keps/sig-cli/0024-kubectl-plugins.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ See https://github.com/kubernetes/kubernetes/issues/53640 and https://github.com
 * Relay all information given to `kubectl` (via command line args) to plugins as-is.
   Plugins receive all arguments and flags provided by users and are responsible for adjusting their behavior
   accordingly.
-* Provide a way to limit which command paths can and cannot be overriddden by plugins in the command tree.
+* Provide a way to limit which command paths can and cannot be overriden by plugins in the command tree.
```
[the remark this answers is not in the document]
Comment (mooncak) [reply keps/sig-cli/0024-kubectl-plugins.md]: > This needs to be `overridden`.
> 
> https://www.dictionary.com/browse/overridden

ok, thanks.

**Pull request merged**